definitions/checks/backup/incremental_parent_type.rb

```diff
@@ -0,0 +1,36 @@
+module Checks::Backup
+  class IncrementalParentType < ForemanMaintain::Check
+    metadata do
+      description 'Check if the incremental backup has the right type'
+      tags :backup
+      param :incremental_dir, 'Path to existing backup directory'
+      param :online_backup, 'Select for online backup', :flag => true, :default => false
+      param :sql_tar, 'Will backup include PostgreSQL tarball', :flag => true, :default => false
+      manual_detection
+    end
+
+    def run
+      return unless @incremental_dir
```

Comment:
This parameter is required right? So can this ever be true?

Reply:
This parameter can be `nil` (as in: set to `nil` by the caller) when we're creating a non-incremental (=base) backup